content/en/blog/2022/frontend-overhaul/index.md

```diff
@@ -64,7 +64,7 @@ covering each of the following OTel-supported languages:
 - [Rust](/docs/instrumentation/rust/)
 
 Every microservice has a specific goal and can communicate with others by using
-a global gRPC definition. Persistent information is saved into a PostgresSQL
+a global gRPC definition. Persistent information is saved into a postgresql
```

Comment:
i think postgres should remain capitalized

Reply:
```suggestion
a global gRPC definition. Persistent information is saved into a postgreSQL
```